lib/NetPacksLib.cpp

```diff
@@ -1167,26 +1167,38 @@ DLL_LINKAGE void BulkMoveArtifacts::applyGs(CGameState * gs)
 
 DLL_LINKAGE void AssembledArtifact::applyGs(CGameState *gs)
 {
-	CArtifactSet *artSet = al.getHolderArtSet();
+	CArtifactSet * artSet = al.getHolderArtSet();
 	const CArtifactInstance *transformedArt = al.getArt();
 	assert(transformedArt);
-	assert(vstd::contains(transformedArt->assemblyPossibilities(artSet), builtArt));
+	bool combineEquipped = true;
+	if(al.slot >= GameConstants::BACKPACK_START)
+		combineEquipped = false;

```

Comment:
Like in other Nordsoft suggestions:
`combineEquipped = (al.slot < GameConstants::BACKPACK_START)`